ush/hpssarch_gen.sh

```diff
@@ -590,5 +590,177 @@ if [ $type = "enkfgdas" -o $type = "enkfgfs" ]; then
 fi   ##end of enkfgdas or enkfgfs
 #-----------------------------------------------------
 
+#-----------------------------------------------------
+if [ $type = "efsoigdas" ]; then
+#-----------------------------------------------------
+
+  #IAUFHRS_ENKF=${IAUFHRS_ENKF:-6}
+  IAUFHRS_ENKF="6,12,18,24,30"
+  lobsdiag_forenkf=${lobsdiag_forenkf:-".false."}
+  nfhrs=`echo $IAUFHRS_ENKF | sed 's/,/ /g'`
+  NMEM_ENKF=${NMEM_ENKF:-80}
+  NMEM_EARCGRP=${NMEM_EARCGRP:-10}               ##number of ens memebers included in each tarball
+  NTARS=$((NMEM_ENKF/NMEM_EARCGRP))
+  [[ $NTARS -eq 0 ]] && NTARS=1
+  [[ $((NTARS*NMEM_EARCGRP)) -lt $NMEM_ENKF ]] && NTARS=$((NTARS+1))
+##NTARS2=$((NTARS/2))  # number of earc groups to include analysis/increments
+  NTARS2=$NTARS
+
+  dirpath="efsoi${CDUMP}.${PDY}/${cyc}/atmos/"
+  dirname="./${dirpath}"
+  head="${CDUMP}.t${cyc}z."
+
+  #..................
+  rm -f efsoi${CDUMP}.txt
+  touch efsoi${CDUMP}.txt
+
+  echo  "${dirname}${head}enkfstat                   " >>efsoi${CDUMP}.txt
+#  echo  "${dirname}${head}gsistat.ensmean            " >>efsoi${CDUMP}.txt
+  echo  "${dirname}osense_${PDY}${cyc}_init.dat      " >>efsoi${CDUMP}.txt
+  echo  "${dirname}${head}abias_int.ensmean          " >>efsoi${CDUMP}.txt
+  if [ -s $ROTDIR/${dirpath}${head}cnvstat.ensmean ]; then
+       echo  "${dirname}${head}cnvstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  if [ -s $ROTDIR/${dirpath}${head}oznstat.ensmean ]; then
+       echo  "${dirname}${head}oznstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  if [ -s $ROTDIR/${dirpath}${head}radstat.ensmean ]; then
+       echo  "${dirname}${head}radstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  for FH in $nfhrs; do  # loop over analysis times in window
+     FHR=$(printf %03i $FH)
+     if [ $FHR -eq 6 ]; then
+        if [ -s $ROTDIR/${dirpath}${head}atmanl.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atmanl.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+	fi
+        if [ -s $ROTDIR/${dirpath}${head}atminc.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atminc.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+     else
+        if [ -s $ROTDIR/${dirpath}${head}atma${FHR}.ensmean${SUFFIX} ]; then
+	    echo  "${dirname}${head}atma00${FHR}.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+        if [ -s $ROTDIR/${dirpath}${head}atmi${FHR}.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atmi00${FHR}.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+     fi 
+  done # loop over FHR
+  for fstep in ecenfsoi esfcfsoi eupdfsoi efcsfsoi eposfsoi ; do
+   echo  "logs/${CDATE}/${CDUMP}${fstep}*.log        " >>efsoi${CDUMP}.txt
```

Comment:
This doesn't look to be indented properly

Reply:
fixed